scalableminds/webknossos
# Add UI to globalize incomplete floodfill(s)

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- set up wk worker locally (use the newest version and let me know in case you want to use docker; I had to do some adjustments to the setup which are not yet on the worker's master)
- open a volume annotation (ideally with a fallback segmentation layer)
- brush something (A)
- initiate a floodfill on a rather large segment (B)
- a bounding box should be added to the bbox tab
- use the new globalize-button. see screenshot:
![image](https://user-images.githubusercontent.com/2486553/150531582-9a2610e1-6496-45ea-9f91-1a26e4aa8b53.png)
- a modal should open where you can specify a new DS name:
![image](https://user-images.githubusercontent.com/2486553/150531737-15bb183f-fd52-4ce3-b9c1-5739ca9238d3.png)
- when the job has completed, a new DS should appear in the dashboard which should have a segmentation layer with:
  - brush (A) should be included
  - the floodfill from (B) should be completed

Before globalize:
![before-globalize](https://user-images.githubusercontent.com/2486553/150534710-21391f3c-740e-4f2a-9445-fb202afc1bf7.png)

After globalize:
![after-globalize](https://user-images.githubusercontent.com/2486553/150534694-2595c054-a7d2-4ceb-9909-922f30bc17eb.png)


### Issues:
- UI for #5763

------
(Please delete unneded items, merge only when none are left open)
